openpype/modules/kitsu/utils/update_op_with_zou.py

```diff
@@ -463,6 +494,14 @@ def sync_project_from_kitsu(dbcon: AvalonMongoDB, project: dict):
     bulk_writes.append(write_project_to_op(project, dbcon))
 
     if project["project_status_name"] == "Closed":
+        kitsu_active_state = KitsuStateToBool[project["project_status_name"]]
```

Comment:
Le souci en l'état est que si un projet Kitsu est ré-ouvert après avoir été `closed` il ne sera pas ré-ouvert côté OpenPype. À voir si il est possible de ré-ouvrir un projet Kitsu `closed` mais même si aujourd'hui la feature n'est pas possible **CG Wire** peuvent ajouter cell-ci dans le futur et donc il faut gérer ce cas.

Reply:
Je ne suis pas sûr de suivre ce que tu veux dire. Il ya deux possibilités :
- Soit le projet Kitsu récupéré est tagué comme `Opened`, et dans ce cas là, c'est la ligne 494 juste au dessus qui traduit le dictionnaire récupéré dans le format accepté par OpenPype via la méthode `write_project_to_op`
- Soit le projet est tagué comme `closed`, et dans ce cas là on rentre dans cette condition qui se contente de mettre à jour l'état actif du projet (si différent de celui enregistré) puis passe à un autre projet.
Il est tout à fait possible côté Kitsu de réouvrir un projet fermé, je l'ai fais de mon côté dans mes tests avec succès.

Reply:
Ah je viens de voir la ligne `"active": project["project_status_name"] != "Closed",` dans la fonction `write_project_to_op`

Reply:
peut être alors remplacer cette ligne par `"active": KitsuStateToBool[project["project_status_name"]],`

Reply:
et dernière chose tout le block `if project["project_status_name"] == "Closed":` devrait être placé avant la ligne `if not project_dict:` (actuellement la 492)